pages/Keiko.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../styles/keiko.css">
+  <title>Keiko's Sass Floof Life</title>
+</head>
+<body>
+<div class="container">
+  <h2 class="ptitle">Keiko</h2>
+  <div id="mySidenav" class="sidenav">
+    <a href="/pages/index.html" id="home">Home</a>
+    <a href="/pages/snoopy.html" id="snoopy">Snoopy</a>
+    <a href="/pages/Bowie.html" id="bowie">Bowie</a>
+    <a href="mailto:abc@example.com" id="contact">Contact</a>
+  </div>
+  </div>
+
+  <div class="gallery">
+    <img src="/images/Keiiko.jpeg" alt="...">
```

Comment:
It's a good idea to give your images alt-text so that folks who can't see the images know what's in them!